packages/core/test/scxml.test.ts

```diff
@@ -348,7 +353,7 @@ interface SCIONTest {
 }
 
 async function runW3TestToCompletion(machine: StateNode): Promise<void> {
-  await new Promise((resolve, reject) => {
+  await new Promise((res, reject) => {
```

Comment:
why the change? :p you have shortened just `resolve` to `res` but at the same time, you have left `reject` untouched.

Reply:
During development there was a `resolve()` function - avoided collision